docs/2-document-guidelines/4-global-audience.md

```diff
@@ -1,27 +1,27 @@
+The process of localizing software has two steps. The first step is when developers provide a mechanism and method for the eventual translation of the product and its interface to suit local preferences and languages for users worldwide. Its process includes designing the product and documentation such that localization requires minimum effort. This process is called internationalization (*i18n*). WordPress developers have done this already, so in theory, WordPress can be used in any language.
 
-The second step is the actual localization (*l10n*). It is the process by which the a product/service is translated and adapted to another language and culture along with its documentation. The framework prescribed by the developers of the software is used for this purpose. Localization is done by people who are familiar with the local language and culture. For example, the *l10n* process involves adapting to the laws, currency and political requirements of a specific locale (market). WordPress has already been localized into many languages (see WordPress' [list of teams](https://make.wordpress.org/polyglots/teams/) for more information).
+The second step is the actual localization (*l10n*). It is the process by which a product/service is translated and adapted to another language and culture along with its documentation. The framework prescribed by developers of the software is used for this purpose. Localization is done by people who are familiar with the local language and culture. For example, the *l10n* process involves adapting to the laws, currency and political requirements of a specific locale (market). WordPress has already been localized into many languages (see WordPress' [list of teams](https://make.wordpress.org/polyglots/teams/) for more information).
 
 Translation is simply changing the language of the content to another language. Translation can be done by both editors from the community as well as machine-aided translation.
 
 ## General guidelines
 
 - If you write for international audiences, research, read, and learn more about them.
 - Don't be specific in terms of culture and religion in your documentation.
-- Use diverse examples that would cater to an international audience. These include, diverse and inclusive names, email addresses, locations, and professions in examples.
-- Avoid colloquialisms, popular culture references, slang, and idioms in your documentation. Phrases like *you got it*, *that's sick!*, *cool* are hard to translate and perceive by global audiences.
```

Comment:
We'll have to use oxford commas in our documentation: https://github.com/WordPress/WordPress-Documentation-Style-Guide/blob/master/docs/4-punctuation/commas.md#oxfordserial-commas

Reply:
Ah ok, let me fix that back.